src/main/java/com/theelixrlabs/healthcare/response/FailureResponse.java

```diff
@@ -0,0 +1,15 @@
+package com.theelixrlabs.healthcare.response;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import java.util.List;
+
+/**
+ * A failure response class to wrap failure API responses.
+ */
+@Data
+@AllArgsConstructor
+public class FailureResponse {
+   private Boolean success;
```

Comment:
Make use of primitive boolean
so the default values can be used

Reply:
I changed to boolean.